routes/index.js

```diff
@@ -76,6 +76,10 @@ var routes = {
     };
     var request_id = request.headers['x-request-id'];
 
+    if (transaction.description === "Thunderbird") {
+      metadata.thunderbird = true;
+    }
```

Comment:
Does accounting take into account monthly subscriptions, where charges don't have a description set?

This method that uses metadata ensures that everytime a charge comes through, this flag is on the customer object, even for monthly subs